src/oca_github_bot/__main__.py

```diff
@@ -41,6 +42,8 @@
 def main():
     # configure logging
     logging.basicConfig(level=logging.DEBUG)
+    # make sure the CLA database is available
+    init_cla_database(config.CLABOT_CACHE)
```

Comment:
Could you move this initialization to the beginning of the task?

Reply:
probably. But this looks like waste of time, as:

1. this will need checking for the file each time the task is invoked
2. probably this will need locking to prevent async invocations of the task from getting in a race condition over the existence of the file. 

Are you sure this is worth it?

Reply:
Hm, yes. Then could you decorate it with `@switchable('clabot')` as well as the cla bot task? So they can be both disabled with the same configuration.